### ISSUE TITLE
training issue 

### ISSUE THREAD

**Issue by Sahilkumawat-01** [BrandenKeck/pytorch_fun#1]
I used your code as a reference to train on my dataset, train folder contains 2000x6 ip tensor and 1000x1 target tensor, it i made proper changes, but while training it is getting saturation from 3rd epoch 
epoch 1 loss = 76.71091390440736
epoch 2 loss = 54.45513159112085
epoch 3 loss = 455133580557906
can u help me with suitable suggestion to avoid this issue ??
